### PR TITLE
ROU-4100: Add new support extensibility parameters to range slider

### PR DESCRIPTION
**Pull request by bmarcelino-fe**
This PR is for adding the new parameters to extensibility configurations.

### Checklist
-   [X] tested locally
-   [X] documented the code
-   [X] clean all warnings and errors of eslint
-   [ ] requires changes in OutSystems (if so, provide a module with changes)
-   [ ] requires new sample page in OutSystems (if so, provide a module with changes)

### CODE REVIEW

**Review by JoaoFerreira-FrontEnd — APPROVED** [review]

File: src/scripts/Providers/OSUI/RangeSlider/NoUISlider/Utils.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace Providers.OSUI.RangeSlider.NoUISlider.Utils {
+	export function SetRangeValues(providerConfigs: ProviderConfigs): ProviderConfigs {
```
Comment (BenOsodrac) [inline src/scripts/Providers/OSUI/RangeSlider/NoUISlider/Utils.ts:3]: I think there might be a Flatpickr.Range type, instead ProviderConfigs

File: src/scripts/Providers/OSUI/RangeSlider/NoUISlider/Utils.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+		const rangeSliderConfigs = JSON.parse(providerConfigs);
```
Comment (BenOsodrac) [inline src/scripts/Providers/OSUI/RangeSlider/NoUISlider/Utils.ts:4]: private const, so it should be _rangeSliderConfigs

File: src/scripts/Providers/OSUI/RangeSlider/NoUISlider/Utils.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+		// Check if the range key is empty. If true, remove from the array
+		if (rangeSliderConfigs.range.length <= 0) {
+			delete rangeSliderConfigs.range;
+		} else {
+			const rangeValues = {};
```
Comment (BenOsodrac) [inline src/scripts/Providers/OSUI/RangeSlider/NoUISlider/Utils.ts:10]: private const again, _

**Review by BenOsodrac — CHANGES_REQUESTED** [review]

File: src/scripts/Providers/OSUI/RangeSlider/NoUISlider/Utils.ts (hunk 3 above)
Comment (bmarcelino-fe) [reply src/scripts/Providers/OSUI/RangeSlider/NoUISlider/Utils.ts:10]: fixed

File: src/scripts/Providers/OSUI/RangeSlider/NoUISlider/Utils.ts (hunk 2 above)
Comment (bmarcelino-fe) [reply src/scripts/Providers/OSUI/RangeSlider/NoUISlider/Utils.ts:4]: fixed

File: src/scripts/Providers/OSUI/RangeSlider/NoUISlider/Utils.ts (hunk 1 above)
Comment (bmarcelino-fe) [reply src/scripts/Providers/OSUI/RangeSlider/NoUISlider/Utils.ts:3]: I'm using the same type as the NormalizeProviderConfigs method. I'm passing all the configs and not only one.

**Review by BenOsodrac — APPROVED** [review]

**Review by JoaoFerreira-FrontEnd — APPROVED** [review]

**Pull request merged**